mdoc/Consts.cs

```diff
@@ -4,6 +4,6 @@ namespace Mono.Documentation
 	public static class Consts
 	{
 		// this is only a placeholder
-		public static string MonoVersion = "5.0.0.7";
+		public static string MonoVersion = "5.0.0.8";
```

Comment:
I was just following the convention used before (mdoc just used the same constants file that was generally used in mono, so that's why the version was pegged to the mono release. However, I'm happy to take suggestions ... should the version # be a supplied parameter to the makefile and not stored in source control?